e2e/wdio/core/tests/slider.e2e-spec.ts

```diff
@@ -202,10 +206,11 @@ describe('slider test suite', function() {
             expect(doesItExist(playgroundExamples + sliderLabels)).toBe(false);
         });
 
-        xit('should check ability to disable slider', () => {
-            expect(getAttributeByName(playgroundExamples + sliderAttr, disabledAttribute)).toBe('false');
+        it('should check ability to disable slider', () => {
```

Comment:
same.. clicking is pointless if there's no assertion. just skip and we'll look harder later